catalog-info.yaml

```diff
@@ -1146,6 +1180,12 @@ spec:
           message: Daily trigger of Iron Bank validation Pipeline per branch
           env:
             PIPELINES_TO_TRIGGER: 'beats-ironbank-validation'
+        Daily run of AWS tests:
+          branch: main
+          cronline: 0 */12 * * *
```

Comment:
this cronline will result in running it twice a day (every 12hrs): given that the name is "Daily" is this the intention?

Reply:
It's intention :)